README.md

```diff
@@ -1 +1,45 @@
-# hw01
+# Emily Young - HW01  
```

Comment:
Hi Emily, reading your introduction, was very simple and easy. You gave a very detailed list, and highlighted some words in the middle which made it all the more fun to read!